metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLFeatureSourceType.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.DatasetUrn
+
+/**
+ * Input source type for MLFeatures and MLPrimaryKeys
+ */
+typeref MLFeatureSourceType = union[DatasetUrn]
```

Comment:
If we want to remove references to specific urn, should we remove this typeref and rely on the relationship annotation?

Reply:
@jjoyce0510

Reply:
Yes - let's make this an Urn